CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing to CICE
+
+Welcome to the CICE Consortium ! First, thanks a lot for considering to contribute to CICE.
+
+There are many ways to contribute, and we appreciate your efforts. For an introductory tour, head over to our [Contribution guide][contributing] and take a look at the [Resources page], our go-to index with links to code, documentation, testing information, guidelines, and more.
```

Comment:
It doesn't appear the resources page is linked. Another thing that @eclare108213 has also suggested in the past is including a full URL instead of just a link. We could consider that too.

Reply:
linked to resources page, did not include full URLs